mapswipe_workers/mapswipe_workers/project_types/arbitrary_geometry/project.py

```diff
@@ -47,6 +47,16 @@ def __init__(self, project_draft: dict) -> None:
         if "TMId" in project_draft.keys():
             self.TMId = project_draft["TMId"]
 
+        self.answerLabels = project_draft.get(
+            "answerLabels",
+            [
+                {"color": "", "label": "no", "value": 0},
+                {"color": "green", "label": "yes", "value": 1},
+                {"color": "orange", "label": "maybe", "value": 2},
+                {"color": "red", "label": "bad imagery", "value": 3},
+            ],
```

Comment:
```suggestion
            [
                {"color": "red", "label": "no", "value": 0},
                {"color": "green", "label": "yes", "value": 1},
                {"color": "grey", "label": "not sure", "value": 2},
            ],
```

Reply:
Yeah, that seems to be the case; I checked with the data of [this](https://mapswipe.org/en/project.html?projectId=-NC5K2TtmuyFBUTdHgJt) footprint project, and 3 seems to be unused.